### ISSUE TITLE
Got errors when I ran A3 and B1

### ISSUE THREAD

**Issue by Haoran-Xue** [mapleforest/HaploMerger2#24]
Dear Shengfeng,

I got expected results when I ran scripts A3 and B1 with the genome bbv18wm. However, I got the same error when I ran A3 and B1 with my own genome:

> ../bin/faDnaPolishing.pl --legalizing --noLeadingN --removeShortSeq=200
> 
> --legalizing=1 .
> --maskShortPortion=0 .
> --noLeadingN=1 .
> --removeShortSeq=200 .
> Problems with the fasta file; no symbol > found !

I'm sure that the entries in my genome file start with ">", like this:
\>1
TTGCACTGGTCTGACTGAATAGTTTCATTTCACCATGAATGACTTTGTGGAGcaacaaaa
gaaatattttgCTGAACATGAacgtttttttttgttattccgAGCTGATTGTCCCGTTGA
TATGCATGCTCTTGTTCAACTCATTTTCactgttattattattattataaaaaaactttt

Do you know why the > symbols in my file are not recognized?

Best regards,
Haoran

**Comment by mapleforest** [comment]
Probably your fasta file is not in unix/linux TXT format. Try to convert them from DOS TXT to unix/linux TXT.

**Comment by Haoran-Xue** [comment]
Yeah, it worked after I converted them from DOS TXT to unix/linux TXT. Thank you so much!
